src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/OffenderVisitBalance.kt

```diff
@@ -13,6 +16,15 @@ data class OffenderVisitBalance(
   @Column(name = "OFFENDER_BOOK_ID", nullable = false)
   val offenderBookingId: Long,
 
+  @OneToOne
+  @JoinColumn(
+    name = "OFFENDER_BOOK_ID",
+    referencedColumnName = "OFFENDER_BOOK_ID",
+    insertable = false,
+    updatable = false
+  )
+  val offenderBooking: OffenderBooking? = null,
+
```

Comment:
Feels like I should be able to combine this with the `@Id` field, but requires another PR to work that out

Reply:
Im surpirsied it doesnt complain that the same col is mapped twice